api/app/signals/apps/search/tasks.py

```diff
@@ -46,3 +47,40 @@ def delete_from_elastic(signal):
         signal_document.delete()
     except NotFoundError:
         log.warning(f'Signal {signal.id} not found in Elasticsearch')
+
+
+@app.task
+def index_signals_updated_in_date_range(from_date, to_date):
+    """
+    Index all Signals updated in the given date range (Copied from the elastic_index management command)
```

Comment:
Please document how to call this from the Django Admin.

Reply:
Just create a new periodic task in the Django admin:

![Screenshot from 2022-10-26 12-40-29](https://user-images.githubusercontent.com/23056530/198006015-df894967-a98d-4acc-803d-3822b73adb57.png)